backend/src/utils/config.ts

```diff
@@ -147,6 +147,8 @@ export interface Config {
       name: string
       image: string
     }>
+    //max model size is calculated in gigabytes
+    maxModelSizeGB: number
```

Comment:
You should also update the `UiConfig` type in `frontend/types/types.ts` to match this.